pkg/lib/cockpit-components-table.jsx

```diff
@@ -41,6 +41,7 @@ import './cockpit-components-table.scss';
  * See https://www.patternfly.org/v4/components/table/
  * Properties (all optional unless specified otherwise):
  * - caption
+ * - id: optional identifier
```

Comment:
Please add this to `ListingTable.propTypes`. One of these days we really need to figure out how to validate propTypes during build or tests..